src/test/java/org/snomed/snowstorm/core/data/services/ConceptServiceTest.java

```diff
@@ -507,6 +507,28 @@ void testSaveConceptWithDescription() throws ServiceException {
 		assertEquals(1, description.getAcceptabilityMapFromLangRefsetMembers().size());
 	}
 
+	@Test
+	void testSaveConceptWithAnnotation() throws ServiceException {
+		final Concept concept = new Concept("50960005", 20020131, true, "900000000000207008", "900000000000074008");
+		concept.addDescription(new Description("84923010", 20020131, true, "900000000000207008", "50960005", "en", FSN,
```

Comment:
@QuyenLy87 can you add a test for a rebasing/conflict scenario? There's quite a few in `BranchMergeServiceTest`. I wonder what will happen if we rebase a concept with an annotation against itself without an annotation. Depending on which column is selected, we shouldn't accidentally lose the annotation.